Decompiler/Options.cs

```diff
@@ -42,6 +42,26 @@ class Options
             HelpText = "Verify checksums and signatures.")]
         public bool VerifyVPKChecksums { get; set; }
 
+        [Option("tool", DefaultValue = false,
+            HelpText = "Enable generic vpk dump tool for all supported resources.")]
+        public bool VPKTool { get; set; }
```

Comment:
Well, first, I had no idea where this tool is used ([here?](https://github.com/SteamDatabase/GameTracking/blob/master/common.sh)), and second, I followed the suggestion to have independent cli option.   
But if it's only used there, then I guess it makes sense to drop the hard-coded options, and add in common.sh: `-e "vxml_c,vjs_c,vcss_c,vsndevts_c,vpcf_c,txt,cfg,res,png,jpg"`